e2e/cypress/support/ui_commands.js

```diff
@@ -432,3 +447,49 @@ Cypress.Commands.add('checkRunLDAPSync', () => {
         }
     });
 });
+
+Cypress.Commands.add('createChannel', (name, isPrivate, purpose, header, isNewSidebar) => {
```

Comment:
Please change to `uiCreateChannel` and move to appropriate file at `/support/ui/` folder. (Not sure if better to create a new `/support/ui/channel.js` so I'll defer to you).

Note that the `ui` prefix was agreed upon to easily identify that a command is done via UI flow and to easily discover via autocomplete/intellisense.

Same comment for other UI commands.